Added typescript definitions

orendin opened the pull request:
- we are using the library in our typescript apps but always have to include our own, separate typescript defintion file.
- now including this into the library itself and created a pull request
- it would be helpful for other people using this with typescript if you could merge this into main/master.

orendin committed:
Added typescript definitions
- we are using the library in our typescript apps but always have to include our own, separate typescript defintion file.
- now including this into the library itself

bsyk commented:
@orendin this is great, thanks for contributing. The returned token object also includes a 'renew_time' property. While most consumers don't use that, would it cause any issue if a typescript consumer wanted to use it and it not be included in the IToken definition?

bsyk commented on predix-uaa-client.d.ts:
```diff
@@ -0,0 +1,11 @@
+    export interface IToken {
+        access_token: string;
+        refresh_token: string;
+        expire_time: string;
```
`expire_time` Is a number (ms since epoch).

orendin committed:
Extended IToken
- Fixed type for expire_time by making it a number
- added renew_time of type number

orendin commented:
You are right on both terms. I have updated the code accordingly and committed the changes.

bsyk commented:
I'll update the version number and publish to npm tomorrow.
